spec/support/api/schemas/teams.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
I suppose this file should also be renamed to `spec/support/api/schemas/datatables/datatables_teams.json`. I mean, if we want to mirror database model as much as possible.